Evaluation documents refinements rework (#86)
koppor/adr-manager: docs/evaluation/material-during-study/meeting-summary-english.md

**Meeting protocol - 12.01.2021, 14:00 to 17:30**

**Recorder**: Anna Mustermann

| **name**        | **present** |
| --------------- | ------------ |
| Anna Mustermann | yes           |
| Bob Musterfrau  | yes           |
| Marco Mayer     | yes          |
| Matt Müller     | no         |

## Meeting Topic

> Use of e2e test frameworks

Reviewer: ```suggestion
Using an end-to-end (e2e) testing framework
```

**Context**

We need an e2e test framework for our application. It needs to support JavaScript. Which test framework can we use?

**Options**

- Anna proposes the use of Cypress tests
- Bob is more in favor of using Selenium tests
- Marco is against e2e tests

**Discussion**

- Cypress
    - Advantages
        - Cypress runs directly in the browser and is fast

>         - It is installed in a short amount of time and no configuration is required

Reviewer: ```suggestion
        - It can be installed quickly and no configuration is required
```
